bundles/CoreBundle/Command/Document/GeneratePagePreviews.php

```diff
@@ -43,9 +44,60 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Prefix for the document path, eg. https://example.com, if not specified, Pimcore will try use the main domain from system settings.',
                 null
+            )
+            ->addOption(
+                'parent',
+                'p',
+                InputOption::VALUE_OPTIONAL,
+                'Only create preview of objects in this folder (ID|path).'
+            )
+            ->addOption(
+                'regex',
+                'r',
+                InputOption::VALUE_OPTIONAL,
+                'All child objects matching the regex pattern will be excluded.'
```

Comment:
```suggestion
                'All documents matching the regex pattern will be excluded.'
```

Reply:
The regex tests the path property, so maybe we should even clarify it further? Something like "Excludes all documents whose path property matches the regex pattern"?
What do you think?

Reply:
yes, even better 👍